source/Calamari/Integration/Iis/InternetInformationServer.cs

```diff
@@ -24,13 +24,15 @@ public bool OverwriteHomeDirectory(string iisWebSiteNameAndVirtualDirectory, str
             var iisSiteName = parts.First();
             var remainder = parts.Skip(1).Where(x => !string.IsNullOrWhiteSpace(x)).ToArray();
             var virtualDirectory = remainder.Length > 0 ? string.Join("/", remainder) : null;
+            WebServerSupport server = new WebServerSevenSupport();
 
-            var server = legacySupport 
-                ? WebServerSupport.Legacy() 
-                : WebServerSupport.AutoDetect();
-
+#if IIS_SUPPORT
```

Comment:
This part of code will change once https://github.com/OctopusDeploy/Calamari/pull/1343 merges to change how we deal with this legacy IIS6 class